src/main/scala/com/fulcrumgenomics/util/SimpleCounter.scala

```diff
@@ -69,6 +62,6 @@ class SimpleCounter[T](implicit tt: TypeTag[T]) extends Iterable[(T, Long)] {
   /** Iterates over all the objects and their counts. */
   override def iterator: Iterator[(T, Long)] = this.counts.entrySet().iterator().map(entry => (entry.getKey, entry.getValue))
 
-  /** Gets sum of counts. */
-  def totalCounts: Long = _totalCounts.toInt
+  /** Gets the number of counts stored in this counter (sum over all count). */
+  def sumOfCounts: Long = _totalCounts.toInt
```

Comment:
how about `totalObservations`?  Or just `total`?  I think the problem with these kinds of methods is finding good names.